packages/neuron-ui/src/components/Settings/actionD.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+export default () => (
```

Comment:
Actions 0 - N should not belong to `Settings`. We'll have the `Settings` component a place for the app Preferences, where we might put many panels for different settings.